frontend/packages/dev-console/src/components/pipelines/pipeline-overview/TriggerLastRunButton.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { connect } from 'react-redux';
+import { impersonateStateToProps } from '@console/internal/reducers/ui';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { useAccessReview } from '@console/internal/components/utils';
+import { Button } from 'patternfly-react';
+import { rerunPipeline } from '../../../utils/pipeline-actions';
+import { PipelineModel } from '../../../models';
+
+type TriggerLastRunButtonProps = {
+  pipeline: K8sResourceKind;
+  disable: boolean;
```

Comment:
This should probably be called `disabled` and should be optional for the purity of the component.

That way the interface is similar to that a button:
```
<TriggerLastRunButton disabled />
<Button disabled />
```